.github/workflows/releaser.yml

```diff
@@ -25,11 +25,22 @@ jobs:
         uses: mukunku/tag-exists-action@9298fbcc409758ba624a0ae16b83df86637cb8ce # v1.2.0
         with:
           tag: ${{ steps.version.outputs.version }}
-      - name: Create release
+      - id: push
+        name: Create tag
         if: steps.version.outputs.version != '' && steps.tag.outputs.exists == 'false' && (
             github.ref == format('refs/heads/{0}', github.event.repository.default_branch) ||
             contains(fromJSON(steps.labels.outputs.labels), 'release')
           )
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          REPOSITORY: ${{ github.repository }}
+          TAG: ${{ steps.version.outputs.version }}
+          SHA: ${{ github.sha }}
```

Comment:
Since `releaser` is triggered on `push` events, the `github.sha` will reference the merge commit here.